### ISSUE TITLE
"Navigator TCP Connection Fail" message after updating to Navigator 19

### ISSUE THREAD

**Issue by myurkoski** [johnjore/Centrafuse-MapFactor-Navigator#2]
Sorry to be the only guy that reports bugs!

Just updated to Navigator 19, and I'm constantly getting a "Navigator TCP Connection Fail" message. It was working great previously, so I'm not sure whats changed between the two. 

The PC had a network connection. I'm running CF 4.4.9 on Windows 10 (with latest updates 1909). Any idea on how to suppress this?

**Comment by johnjore** [comment]
Message is by design when it can't connect to Mapfactor Navigator. Probably you only need to enable the TCP connection in Navigator.


It's required for the plugin to work under CF.


________________________________

**Comment by myurkoski** [comment]
Well I looked all over for such a setting, but I couldn't find one, so I launched CF again and mercifully, my PC was slow enough that the initial splash allowed me enough time to select language for interface and voice and accept the warning. Now I no longer receive this warning. 

I'm not recalling having to do this before in 18, so was there something i did that was of similar effect, or is this a new behavior in 19?

**Comment by johnjore** [comment]
This is documented on the first page and is for all Navigator versions. Must complete first run settings before using it under CF.


________________________________

**Comment by myurkoski** [comment]
Ahhh, thank you. I had omitted this because I merely upgraded from 18 to 19, but it appears they are entirely different, with new install locations and everything. Whats funny is that it immediately asked me to find the new exe which I pointed it at, but that didn't prompt me to do the re-setup. I loaded up Nav 19 **outside** of CF and got it all squared away, but apparently when CF runs it it carries its own settings/options so it ignored that setup. CF must be launching it as a different user or something making it treat the CF version separately from a user directly launching Nav19 from their desktop. 

Closing ticket.
